app/services/cleaners/litigator_hardship_claim_cleaner.rb

```diff
@@ -0,0 +1,22 @@
+module Cleaners
+  class LitigatorHardshipClaimCleaner < BaseClaimCleaner
+    private
+
+    def clear_inapplicable_fields
+      clear_ppe if case_type.present? && ppe_not_required?
+    end
+
+    def ppe_not_required?
+      claim.case_stage.unique_code.eql?('NOPTPHNOPPE') && hardship_fee.present?
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/4252/files#r720222724)